### DISCUSSION TITLE
Request: 削除されたユーザーのページを移動または削除したい

### DISCUSSION THREAD

**Discussion by eyelly** [growilabs/growi#6467] [Ideas / Requests]
退社のようなイベント後に利用しなくなったユーザーのページを移動または削除したいです。
対象ページ：/user/<対象ユーザー名>/*

現在は利用しなくなったユーザーで管理者がログインし公開範囲を「自分だけ」とすることで、一般のユーザーからは見えないようにしています。
DBから削除することでユーザーページを削除することは可能かもしれませんが、利便性を考え管理者がGUI上でユーザーページの移動または削除を行いたいです。

**Comment by yuki-takei** [comment]
こちら実装中です。

https://redmine.weseek.co.jp/issues/105325

**Comment by jam411** [comment]
- [v6.2.0](https://github.com/weseek/growi/tree/v6.2.0) でユーザー新規登録時に同じ username のユーザーホームページがある場合はそれを一度削除して作り変えるように仕様変更されました。
また、ユーザー削除時にユーザーホームページも一緒に削除する設定ができるようになりました。
詳しくは[アップグレードガイド](https://docs.growi.org/ja/admin-guide/upgrading/62x.html#growi-v6-2-x-%E3%81%B8%E3%81%AE%E3%82%A2%E3%83%83%E3%83%95%E3%82%9A%E3%82%AF%E3%82%99%E3%83%AC%E3%83%BC%E3%83%88%E3%82%99)をご覧ください。
- [v6.2.4](https://github.com/weseek/growi/tree/v6.2.4) で、設定により削除済みユーザーのユーザーホームページを通常ページと同じように削除できるようになりました。
